test/libsolidity/syntaxTests/functionTypes/comparison_operator_for_external_functions_with_extra_gas_slots.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function external_test_function() external {}
+    function comparison_operator_for_external_function_with_extra_slots() external returns (bool) {
+        return this.external_test_function{gas: 4} == this.external_test_function;
```

Comment:
We could also test that ``this.external_test_function{gas: 4} == this.external_test_function{gas: 4}`` yields another error.
Since the error is not fatal you could just add it to the same test.

Reply:
Added in the latest push